k8s-helm-http-wrapper/helm-wrapper.js

```diff
@@ -2,14 +2,27 @@
 const util = require('util');
 const exec = util.promisify(require('child_process').exec);
 
-
 class HelmWrapper {
+    
     constructor() {
+        this.initialized = false;
         this.helmBinaryLocation = process.env.HELM_BINARY;
-        exec(this.helmBinaryLocation + ' init --service-account tiller');
+    }
+
+    async init() {
+      if (this.initialized == true) {
```

Comment:
Everytime a new class is instantiated, this.initialized will be `false` and helm init will run again.
It should be placed outside of the class, so it can run only once, when the file is loaded to the memory.